l10n/Makefile

```
@@ -0,0 +1,83 @@
+
+all: translate
+
+# This target runs the full translation pipeline:
+# 1. Extracts strings from source code into a .pot template.
+# 2. Merges the template into language .po files.
+# 3. Extracts all untranslated strings.
+# 4. Runs the AI translation script on them.
+# 5. Merges the new translations back into the .po files.
+# 6. Checks the syntax of the final .po files.
+# 7. Cleans up all intermediate files.
+translate:
```

checkmake (0.2.2)

[warning] Target body for "translate" exceeds allowed length of 5 (15).

(maxbodylength)